client/src/components/auth/SignIn.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+import Link from '@mui/material/Link';
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Checkbox from '@mui/material/Checkbox';
+import Box from '@mui/material/Box';
+import Container from '@mui/material/Container';
+
+export default function SignIn() {
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const data = new FormData(event.currentTarget);
+    // eslint-disable-next-line no-console
```

Comment:
Look into eslint

Reply:
Whoops, comment didn't post before I resolved

Reply:
It is best practice to avoid using methods on console, stated in eslint docs [here](https://eslint.org/docs/rules/no-console). I will leave this in for now and it will be replaced with API calls later when the auth functionality is implemented.

Reply:
So yes it is known to not use console logs in a production application. The thing is I believe you should disable the rule globally while in development, so you can re-enable the rule later and it will show you where every console is. These inline rules always override globals, and are harder to find later

Reply:
Yes that makes sense. Will do.